### ISSUE TITLE
exported vars can be exported several times

### ISSUE THREAD

**Issue by SuperUserNameMan** [godotengine/godot#6566]
**Operating system or device - Godot version:**
current head, 64bits, win10

**Issue description** (what happened, and what was expected):

like the title say, a variable can be exported several times without complaints from the editor.

all of the export will appear into the inspector.

if you modify one of them, only the first one will have the "reset" icon, and only the first one will be reset if you press it.

**Steps to reproduce:**

```
export var test = "test";
export var test = "test";
export var test = "test";
export var test = "test";
export var test = "test";
export var test = "test";
```

**Link to minimal example project** (optional but very welcome):

**Comment by Noshyaar** [comment]
Still reproducible/relevant in 3.1 master 9ad1800

**Comment by vnen** [comment]
Not sure if it was fixed in my PR (#19264) but in any case it is fixed in master.
